submissions/tosinIshola/HTML/Ecommerce.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>ALL THINGS FASHION</title>
+</head>
+<body>
+<header>
+    <h1><strong>ALL THINGS FASHION (A.S.F)</strong></h1>
```

Comment:
Hello Tosin,

Your indentation is nice!
You could, however, remove the strong tag within the heading tag — here and anywhere else in the code blocks. Headings are, by default, strong. If you rather wish to increase the font weight, then it’s CSS to the rescue!

Reply:
Okay